packages/block-library/src/comments/index.php

```diff
@@ -76,6 +76,7 @@ function render_block_core_comments( $attributes, $content, $block ) {
 	 * why they are not defined in `block.json`.
 	 */
 	wp_enqueue_script( 'comment-reply' );
+	wp_script_add_data( 'comment-reply', 'strategy', 'defer' );
```

Comment:
Could be considered a nit-pick, but since this is a core script and is only being _enqueued_ here, I think it would be more appropriate to leave this change out of here and instead set the strategy only in WP core, when _registering_ the script.

Reply:
The thinking here was to get this change in Gutenberg to test its impact. Agreed that it would not be relevant for core merge.

Reply:
Added core merge notes in ffd9e5cc6ab